docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM jenkins/jenkins:2.332.3-lts-jdk8
+FROM jenkins/jenkins:2.346.3-lts-jdk8
 LABEL maintainer="OpenSearch"
 ENV JAVA_OPTS -Djenkins.install.runSetupWizard=false
 COPY plugins.txt plugins.txt
-RUN /usr/local/bin/install-plugins.sh < plugins.txt
```

Comment:
So we dont need to run this script anymore?
Or the script has been renamed?
Thanks.

Reply:
This script has been deprecated. Hence replacing with the `jenkins-plugin-cli` https://www.jenkins.io/doc/book/installing/docker/
Tested with the same. Works well.

Reply:
Thanks.